.dockerignore

```diff
@@ -1,2 +1,9 @@
 doc
 **/target
+docker/
+**/*.md
+**/*.txt
+.git/
```

Comment:
We are extracting the git commit in the build process.

Reply:
Good point. I'll have to look into a way to work around that; I think that saving ~65mb of context is worth updating the mechanism by which we get the commit.

Reply:
Though it's worth noting that even excluding `.git`, dockerized images appear to build properly:

```sh
~/paritytech/cumulus$ docker run --rm polkadot:prgn-dockerfile-caching /usr/local/bin/polkadot --version
polkadot 0.7.17-x86_64-linux-gnu
```

Reply:
Yeah I know, we support building without a git ref. I think we just print a warning at compile time.